plane/source/docs/advanced-failsafe-configuration.rst

```diff
@@ -262,6 +262,13 @@ the aircraft is a danger to people or other aircraft. To force an
 immediate termination you should use the :ref:`AFS_TERMINATE<AFS_TERMINATE>` parameter. By
 setting that parameter to 1 the aircraft will immediately terminate.
 
+AFS_OPTIONS
+------------------
+In addition, the behavior of AFS can be modified by the setting of the :ref:`AFS_OPTIONS<AFS_OPTIONS>` bitmask parameter (no bits are set by default):
+
+- bit 0 (+1), if set, the aircraft will continue the mission even after data link is recovered. If not set, the aircraft will return to the mission item right before GCS failsafe occurs 
+- bit 1 (+2), if set, will force entering AUTO when GCS failsafe occurs. This is to make sure AFS is entered if the Plane is flying in other autonomous modes (such as GUIDED).
```

Comment:
so currently if in GUIDED nothing happens on an AFS event? what other "autonomous" modes are there?

Reply:
That's correct. By default, AFS will not trigger on GUIDED.

I'm not sure if I want to list out all the modes. I kind of want to keep the description as it is. This is because autonomous modes might refer to different modes for Plane, Copter, and Rover.  In the code, it is whatever mode is interpreted as AP_AdvancedFailsafe::AFS_AUTO.

Reply:
unfotunately, there is no place in the wiki that designates a vehicles modes as autonomous or non-autonomous, so we will have to list them here since this is the only place in the wiki that this designation is mentioned associated with a mode...even I am uncertain which modes would fall under this category....for example in Plane is LOITER??? you cant say that autonomous is a mode where pilot input is denied, because you can stick mix control even in AUTO....so how do you designate "autonomous"?

Reply:
discussed with Tridge....its not autonomous....its auto-throttle modes in Plane..using AFS in other vehicles is something that is possible but only for an advanced user compiling their own code...so the wiki article will remain Plane focussed for now....I will update this PR accordingly...thanks